bitbot/markov.go

```diff
@@ -38,6 +39,35 @@ var MarkovResponseTrigger = NamedTrigger{
 	},
 }
 
+var MarkovInitTrigger = NamedTrigger{
+	ID:   "markovInit",
+	Help: "Resets markov chain to a fresh chain, or bootstraps it with sample texts. Usage: !markov reset, !markov init",
+	Condition: func(irc *hbot.Bot, m *hbot.Message) bool {
+		return m.Command == "PRIVMSG" && strings.HasPrefix(m.Content, "!markov ")
+	},
+	Action: func(irc *hbot.Bot, m *hbot.Message) bool {
+		cmd := strings.Split(m.Content, " ")
+		if len(cmd) < 2 {
+			return false
```

Comment:
maybe we should print some help text here
```suggestion
                         irc.Reply(m, MarkovInitTrigger.Help)
			return true
```